### DISCUSSION TITLE
osu! lazer multiplayer mod hard rock (no display of changed ar, cs?)

### DISCUSSION THREAD

**Discussion by ScarletSea** [ppy/osu#23019] [Q&A] [answered]
Hey guys,

I have a question regarding osu! lazer multiplayer. 
Let's say I create a multiplayer room for me and my friends. Each of us will add a map with all free mods activated. After doing that I'm able to choose hard rock for my own playthrough. Is it possible for me and everyone who actually chose hard rock to see, how that mod affects the map (AR, CS, star difficulty...) in the lobby? I don't know, if I'm just overseeing something or if it's not possible. 
Can someone enlighten me, pls? :)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/11306.
